### PR TITLE
[WIP]💄 Config eslint and prettier

### PR DESCRIPTION
**Pull request by tung-eh**
Closes https://github.com/canhnd58/achromevement/issues/1

## Proposed Changes

  - Add prettier
  - Update .eslintrc
  -

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,16 +27,22 @@
...
+    "babel-eslint": "^10.0.1",
     "babel-jest": "^23.6.0",
     "babelify": "^10.0.0",
     "browserify": "^16.2.3",
     "chalk": "^2.4.1",
     "del": "^3.0.0",
     "eslint": "^5.7.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-prettier": "^3.1.0",
     "eslint-config-standard": "^12.0.0",
```
Comment (canhnd58) [inline package.json:39]: Remove dependency if no longer used

**Review by canhnd58 — CHANGES_REQUESTED** [review]

**Review by canhnd58 — CHANGES_REQUESTED** [review]
Update doc/coding_style.md to specify how to format files or which command to run before submitting codes.

**Comment by tung-eh** [comment]
Close due to duplicate

**Comment by canhnd58** [comment]
Create a feature request if you still want this to be included

**Pull request closed without merging**